djpyfs/tests.py

```diff
@@ -217,10 +227,11 @@ def test_patch_fs(self):
         Simple check to make sure the filesystem is patched as expected.
         """
         fs = djpyfs.get_filesystem(self.namespace)
-        self.assertTrue(callable(getattr(fs, 'expire')))
-        self.assertTrue(callable(getattr(fs, 'get_url')))
+        self.assertTrue(callable(getattr(fs, 'expire')))   # pylint: disable=literal-used-as-attribute
+        self.assertTrue(callable(getattr(fs, 'get_url')))  # pylint: disable=literal-used-as-attribute
 
 
+# pylint: disable=test-inherits-tests; literal-used-as-attribute
```

Comment:
My understanding is that a comment [placed like this](https://pylint.readthedocs.io/en/latest/user_guide/message-control.html) applies to the rest of the scope it's found in (so in this case, all the code in the rest of this module).  If you just want to disable one instance of a warning, the comment has to go at the end of the line that triggers it.  Mentioning this because you repeat the first disable a couple of times below, which should have no effect.